lib/ansible/plugins/netconf/__init__.py

```diff
@@ -162,7 +168,8 @@ def unlock(self, *args, **kwargs):
     def discard_changes(self, *args, **kwargs):
         """Revert the candidate configuration to the currently running configuration.
         Any uncommitted changes are discarded."""
-        return self.m.discard_changes(*args, **kwargs).data_xml
+        # return self.m.discard_changes(*args, **kwargs).data_xml
```

Comment:
commented out code?

Reply:
At multiple places in the base class methods, I commented out the code, while keeping it there as a reminder to restore it when all the platforms support `.data_xml` and namespace removal. I can take them off and write a `TODO` somewhere if that looks good?

Reply:
Maybe add
`#TODO Restore when all platforms support÷.data_xml`
Then have the commented out code?

Reply:
Agree